shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1053,14 +1077,23 @@ - (void)setMarkedText:(NSString*)markedText selectedRange:(NSRange)markedSelecte
   [self setSelectedTextRangeLocal:[FlutterTextRange
                                       rangeWithNSRange:[self clampSelection:selectedRange
                                                                     forText:self.text]]];
-  [self updateEditingState];
+  if (_enableDeltaModel) {
+    [self updateEditingStateWithDelta];
+  } else {
+    [self updateEditingState];
+  }
 }
 
 - (void)unmarkText {
   if (!self.markedTextRange)
     return;
   self.markedTextRange = nil;
-  [self updateEditingState];
+  if (_enableDeltaModel) {
+    _currentTextEditingDelta = [[FlutterTextEditingDelta alloc] initWithNonText:self.text];
```

Comment:
release the old `_currentTextEditingDelta` I think?

Reply:
If I am now using the setter instead of setting ivar directly, this is not necessary right?